numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -3710,6 +3730,65 @@ static void
     npy_intp i;
     @type@ max_val = 0, min_val = 0;
 
+#if HAVE_ATTRIBUTE_TARGET_AVX2 && @type@==npy_int && NPY_BITSOF_@type@ == 32 
```

Comment:
This will make this miss `LONG_fastclip`, which is likely to be 32 bits on some platforms. Would be better to add a `isint` flag to match the `isfloat` one above, and switch on that - you don't care that it's an `npy_int`, only that it is some kind of int